container/root/etc/php5/fpm/conf.d/web.user.ini

```diff
@@ -3,6 +3,7 @@
 
 memory_limit = 192M
 max_execution_time = 60
+request_terminate_timeout=60
```

Comment:
For now this is probably fine but we may want to chop down both the `max_execution_time` and `request_terminate_timeout` in the future. As we discussed, the clock starts from the client request and if we keep these all at a minute it degrades the client experience.